### PR TITLE
[RHACS] 4.2 Release notes

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://issues.redhat.com/browse/ROX-19148

Preview: https://63889--docspreview.netlify.app/openshift-acs/latest/release_notes/42-release-notes

### CODE REVIEW

File: release_notes/42-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+
+:FeatureName: Vulnerability Management 2.0
+include::snippets/technology-preview.adoc[]
+
+[id="report-configuration-without-an-associated-collection"]
+==== Report configuration without an associated collection
+
+In version 4.0, {product-title-short} released the collections feature that replaced access scopes used in report configurations. {product-title-short} automatically created equivalent collections for access scopes used in existing report configurations and migrated report configurations to use newly-created collections. If the migration failed, the report configurations became non-functional, and {product-title-short} logged the error messages in Central logs and in the {product-title-short} web portal.
+{product-title-short} 4.2 deletes the report configurations that could not be migrated. To avoid losing any existing report configurations, create valid collections for reports that failed the migration and attach the collection to the report configuration before you upgrade to {product-title-short} 4.2.
```
Comment (ShaunaDiaz) [inline release_notes/42-release-notes.adoc:360]: Looks like you have a break here, but it shows up all as one paragraph in the preview. Maybe insert an extra line space?

File: release_notes/42-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+[id="use-container-iam-role-option-in-ecr-integration-for-rhacs-cloud-service"]
+==== Use Container IAM role option in ECR integration for {product-title-managed-short}
+Red Hat has removed the *Use Container IAM role* option from {product-title-managed-short} from the ECR integration configuration page.
+
+[id="notice-for-upcoming-changes"]
+== Notice for upcoming changes
+
+* Previously, the Syslog notifier sent an incorrect message header; {product-title-short} flipped the severity and name fields. When configuring the notifier, you can now select the following options:
+** `CEF`: The corrected message header. Default option when you are configuring using the {product-title-short} portal.
+** `CEF (legacy field order)`: This is the previous incorrect message header. Default option if you use API and do not specify a value for the message header. Red Hat will remove the wrong message header option `CEF (legacy field order)` in version 4.4. Additionally, all legacy field order notifiers will be migrated to `CEF`.
```
Comment (ShaunaDiaz) [inline release_notes/42-release-notes.adoc:376]:
```suggestion
** `CEF (legacy field order)`: This is the previous incorrect message header. Default option if you use API and do not specify a value for the message header. Red Hat plans to remove the wrong message header option `CEF (legacy field order)` in version 4.4. Additionally, all legacy field order notifiers will be migrated to `CEF`.
```

File: release_notes/42-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,435 @@
...
+== Bug fixes
+
+[id="resolved-in-version-420_{context}"]
+=== Resolved in version 4.2.0
+
+*Release date*: 18 Sep 2023
+
+* Previously, if a non-blocking socket failed to open a connection, Collector would still report that as a connection. This issue has been fixed. (ROX-17486)
+* In {product-title-short} 4.2, Collector correctly handles asynchronous connection establishment.(ROX-17486)
+* Operator installations broke if OpenShift cluster-wide proxy was enabled and Central or SecuredCluster CR configured an egress proxy environment variable. This issue is fixed. (ROX-18477)
```
Comment (ShaunaDiaz) [inline release_notes/42-release-notes.adoc:403]:
```suggestion
* Previously, Operator installations broke if the OpenShift cluster-wide proxy was enabled and Central or SecuredCluster CR configured an egress proxy environment variable. This issue is fixed. (ROX-18477)
```
Spell out "CR?" I don't know the acronym; not sure if your readers should.

**Review by ShaunaDiaz — COMMENTED** [review]
Excellent large piece of work! Just a few copy edits and suggestions. As always, your choice and judgment prevail.

File: release_notes/42-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,435 @@
...
+* xref:../release_notes/42-release-notes.adoc#the-core-bpf-collection-method-is-now-ga_release-notes-42[The CORE BPF collection method is now GA]
+* xref:../release_notes/42-release-notes.adoc#rhacs-product-usage-report_release-notes-42[{product-title-short} Product usage report]
+* xref:../release_notes/42-release-notes.adoc#performance-improvements-for-the-compliance-dashboard_release-notes-42[Performance improvements for the Compliance dashboard]
+
+*Vulnerability management*
+
+* xref:../release_notes/42-release-notes.adoc#vulnerability-scanning-support-for-registry-mirrors-in-ocp_release-notes-42[Vulnerability scanning support for Registry Mirrors in {ocp}]
+* xref:../release_notes/42-release-notes.adoc#configure-delegated-image-scanning-in-the-rhacs-portal_release-notes-42[Configure delegated image scanning in the {product-title-short} portal]
+* xref:../release_notes/42-release-notes.adoc#define-new-system-policies-using-cve-age-or-fixability_release-notes-42[Define new system policies using CVE age or fixability]
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply release_notes/42-release-notes.adoc:38]: CVE's are common knowledge for this product audience. So I think it is okay to use it.

File: release_notes/42-release-notes.adoc [hunk 5]
```diff
@@ -0,0 +1,435 @@
...
+
+[id="bug-fixes_{context}"]
+== Bug fixes
+
+[id="resolved-in-version-420_{context}"]
+=== Resolved in version 4.2.0
+
+*Release date*: 18 Sep 2023
+
+* Previously, if a non-blocking socket failed to open a connection, Collector would still report that as a connection. This issue has been fixed. (ROX-17486)
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply release_notes/42-release-notes.adoc:401]: RHACS Jira is not public so we don't usually link them.

File: release_notes/42-release-notes.adoc [hunk 6]
```diff
@@ -0,0 +1,435 @@
...
+* The *Component Upgrade* column in vulnerability reports is renamed to *CVE Fixed In*.
+* Previously, access to the `/api/docs/swagger` API required read permission for the *Integration* resource. Now, it only requires users to be authenticated to access it.
+* Scanner now selects the image for scanning based on its architecture alignment with the Scanner's architecture, rather than consistently choosing `amd64` for multi-arch image scanning.
+** For example, if Scanner runs on `s390x` and an `s390x` version of the multi-arch image exists, it scans that `s390x` image.
+** If no image matches Scanner's architecture, then it attempts to scan the `amd64` version, as it did previously.
+* Telemetry data collection is enabled by default, except for the installations with the offline mode enabled.
+* Integration with {ocp} monitoring is configured and enabled by default for {product-title-short} installations on OpenShift 4. To deactivate this integration, set the flag `monitoring.openshift.enabled` to `false`.
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply release_notes/42-release-notes.adoc:198]: YES

**Comment by gaurav-nelson** [comment]
**praise:** 👏 Thank you @ShaunaDiaz for reviewing and leaving such thoughtful improvement comments. I've addressed all of them.

**Pull request merged**